docs/api-reference/json/json-converter.md

```diff
@@ -83,6 +83,55 @@ will replace the layers descriptor with
 
 Whenever the `JSONConverter` component finds a "type" field, it looks into a "class catalog". This can be a layer, a view, or other objects.
 
+### Functions Catalogs
+
+A map of functions that should be made available to the JSON function resolver. For example, when this configuration is passed to `JSONConverter`:
+
+```js
+function calculateRadius({base, exponent}) {
+  return Math.pow(base, exponenet);
+}
+
+const configuration = {
+  ...,
+  functions: Object.assign({calculateRadius})
```

Comment:
What is the purpose of `Object.assign`?

Reply:
We can set this like `functions: {calculateRadius}`